### DISCUSSION TITLE
Table Editor Not Loading

### DISCUSSION THREAD

**Discussion by ghost** [supabase/supabase#2192] [Questions] [answered]
Hi 👋

Has anyone had issues with the table editor loading nowadays? Mine just seems to spin but never generate. Even on old databases where I haven't changed the configuration it's no longer working. 

Thank you for your help!

**Comment by Danieltech99** [comment, marked as answer]
I have been having the same issue yesterday and today. For me it is only on new projects (my old project works fine). I have tried deleting the project and recreating it multiple times and same issue. What I am most concerned about is the API (accessed through the Supabase Javascript Library) was not working properly either for these projects.

I have not found a solution. Can anyone else help us?

**Reply by ghost** [reply]
Hey Daniel,

I created an issue: https://github.com/supabase/supabase/issues/2198

Good to know it's happening to others. Let me know if you figure anything out and I'll keep trying 👍

**Comment by sumchans** [comment]
I have been having this same issue for some time, also I get this message when I click on the API button 
![image](https://user-images.githubusercontent.com/33538243/124331782-cee05700-db44-11eb-84b0-8ad223b73f35.png)

**Reply by its-danny** [reply]
@sumchans try turning off any "advanced security" features your router has. in my case the issue was related to [this](https://github.com/supabase/supabase/issues/2096#issuecomment-872351970).

**Reply by sumchans** [reply]
Perfect, turning off advanced network security on my router worked. Thanks its-danny.
